packages/boot/src/types.ts

```diff
@@ -159,3 +159,7 @@ export function booter(artifactNamespace: string, ...specs: BindingSpec[]) {
     ...specs,
   );
 }
+
+export interface InstanceWithBooters {
```

Comment:
could you add some API documentation for this interface?

Reply:
:+1:  done